pkg/services/contract/contract.go

```diff
@@ -110,8 +110,8 @@ var ErrMissingPublicURL = errors.New("missing publicUrl")
 
 func (s *service) configureContracts() (err error) {
 	if s.config.ActingPartyCn == "" {
-		err = ErrMissingActingParty
-		return
+		// todo remove this check in 0.17
+		logrus.Info("no actingPartyCn configured, this is needed for v2 contracts (deprecated)")
```

Comment:
"No actingPartyCn configured, this is needed for validation of v2 contracts."

I also think the "deprecated" notice should be logged when ActingPartyCN IS configured? Since they're the parties who might be still depending on v2?

Reply:
added a warning: `actingPartyCn is deprecated, please migrate to v3 contracts and remove the config parameter`